### DISCUSSION TITLE
Add File Explorer and Unique IP Address for Each RDP

### DISCUSSION THREAD

**Discussion by rmeurer007** [blueedgetechno/win11React#59] [Q&A]
Can you Add File Explorer and Unique IP Address for Each RDP?

**Comment by blueedgetechno** [comment]
I can certainly add File explorer but it will be worked upon after completing spotify (50% done). But I didn't understand what do you mean by "Add unique ip" for each RDP. Can you please elaborate?

**Comment by luni-moon** [comment]
> Can you Add File Explorer and Unique IP Address for Each RDP?

Quick question, what is an, "RDP"?

**Reply by blueedgetechno** [reply]
Quick question why is RDP? (RDP is remote desktop protocol, it is used to access remote desktop or application on the same network)

**Reply by luni-moon** [reply]
> Quick question why is RDP? (RDP is remote desktop protocol, it is used to access remote desktop or application on the same network)

Ok, thanks. Quick question, when is RDP?

**Reply by blueedgetechno** [reply]
2006! Quick question, How is RDP?

**Reply by luni-moon** [reply]
> 2006! Quick question, How is RDP?

Ah, you got me there, not too sure, since I didn't know what it is until you told me ;)
Anyways, Who is RDP? (This is getting ridiculous lol).
